cfme/configure/access_control.py

```diff
@@ -168,149 +238,298 @@ def description(self):
+    password = Input(name='password')
+
+    tag = SummaryFormItem('Smart Management', 'My Company Tags')
+
+    cancel_button = Button('Cancel')
+    retrieve_button = Button('Retrieve')
+
+    @View.nested
+    class my_company_tags(Tab):     # noqa
+        TAB_NAME = "My Company Tags"
+        tag_tree = CheckableBootstrapTreeview('tags_treebox')
+
+    @View.nested
+    class hosts_and_clusters(Tab):      # noqa
+        TAB_NAME = "Hosts & Clusters"
+        hosts_clusters_tree = CheckableBootstrapTreeview('hac_treebox')
+
+    @View.nested
+    class vms_and_templates(Tab):       # noqa
+        TAB_NAME = "VMs & Templates"
+        vms_templates_tree = CheckableBootstrapTreeview('vat_treebox')
+
+
+class AddGroupView(GroupForm):
+    add_button = Button("Add")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_configuration and self.access_control.is_opened and
+            self.title.text == "Adding a new Group"
+        )
+
+
+class DetailsGroupView(ConfigurationView):
+    title = Text('#explorer_title_text')
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_configuration and self.access_control.is_opened and
+            self.title.text == 'EVM Group "{}"'.format(self.context['object'].description)
+        )
+
+
+class EditGroupView(GroupForm):
+    save_button = Button("Save")
+    reset_button = Button('Reset')
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_configuration and self.access_control.is_opened and
+            self.title.text == 'Editing Group "{}"'.format(self.context['object'].description)
+        )
+
+
+class AllGroupView(ConfigurationView):
+    title = Text('#explorer_title_text')
+
+    table = Table("//div[@id='main_div']//table")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_configuration and self.access_control.is_opened and
+            self.title.text == 'Access Control EVM Groups'
+        )
+
+
+class EditGroupSequenceView(ConfigurationView):
+    title = Text('#explorer_title_text')
+
+    group_order_selector = UpDownSelect(
+        '#seq_fields',
+        './/a[@title="Move selected fields up"]/img',
+        './/a[@title="Move selected fields down"]/img')
+
+    save_button = Button('Save')
+    reset_button = Button('Reset')
+    cancel_button = Button('Cancel')
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_configuration and self.access_control.is_opened and
+            self.title.text == "Editing Sequence of User Groups"
+        )
+
+
+class GroupEditTagsView(ConfigurationView):
+    title = Text('#explorer_title_text')
+
+    tag_table = Table("//div[@id='assignments_div']//table")
+
+    select_tag = BootstrapSelect(id='tag_cat')
+    select_value = BootstrapSelect(id='tag_add')
+
+    save_button = Button('Save')
+    cancel_button = Button('Cancel')
+    reset_button = Button('Reset')
```

Comment:
Here we do not use GroupForm class as GroupEditTagsView doesn't have all GroupForm elements